avocado-setup.py

```diff
@@ -266,6 +267,18 @@ def install_optional_plugin(plugin):
         # plugin already installed
         pass
 
+def configure_prepostscript(repos, scripttype):
+    i=0
+    cmd= 'mkdir -p /etc/avocado/scripts/job/%s.d' % scripttype
+    helper.runcmd(cmd, ignore_status=True, info_str="%s script directory created" % scripttype)
+    for repo in repos:
+        i=i+1
+        path= "/etc/avocado/scripts/job/%s.d/%sjobscript_repo%d.sh" %(scripttype,scripttype , i)
+        cmd="wget -O %s  %s;chmod 777 %s;%s" % (path, repo, path, path)
```

Comment:
Why do we want to control the script name here ? Let us use whatever script name we have downloaded ?

Reply:
ok but I make that as we can have our control while executing as getting exact name can be tricky

Reply:
we need not worry about names, only placing in a folder that avocado understands is the main work to do here.